docs/source/usersguide/basics.rst

```diff
@@ -0,0 +1,213 @@
+.. _usersguide_basics:
+
+======================
+Basics of Using OpenMC
+======================
+
+----------------
+Creating a Model
+----------------
+
+When you build and install OpenMC, you will have an :ref:`scripts_openmc`
+executable on your system. When you run ``openmc``, the first thing it will do
+is look for a set of XML_ files that describe the model you want to
+simulation. Three of these files are required and another three are optional, as
+described below.
+
+.. admonition:: Required
+   :class: error
+
+   :ref:`io_materials`
+     This file describes what materials are present in the problem and what they
+     are composed of. Additionally, it indicates where OpenMC should look for a
+     cross section library.
+
+   :ref:`io_geometry`
+     This file describes how the materials defined in ``materials.xml`` occupy
+     regions of space. Physical volumes are defined using constructive solid
+     geometry, described in detail in :ref:`usersguide_geometry`.
+
+   :ref:`io_settings`
+     This file indicates what mode OpenMC should be run in, how many particles
+     to simulate, the source definition, and a whole host of miscellaneous
+     options.
+
+.. admonition:: Optional
+   :class: note
+
+   :ref:`io_tallies`
+     This file describes what physical quantities should be tallied during the
+     simulation (fluxes, reaction rates, currents, etc.).
+
+   :ref:`io_plots`
+     This file gives specifications for producing slice or voxel plots of the
+     geometry.
+
+   :ref:`io_cmfd`
+     This file specifies execution parameters for coarse mesh finite difference
+     (CMFD) acceleration.
+
+eXtensible Markup Language (XML)
+--------------------------------
+
+Unlike many other Monte Carlo codes which use an arbitrary-format ASCII file
+with "cards" to specify a particular geometry, materials, and associated run
+settings, the input files for OpenMC are structured in a set of XML_ files. XML,
+which stands for eXtensible Markup Language, is a simple format that allows data
+to be exchanged efficiently between different programs and interfaces.
+
+Anyone who has ever seen webpages written in HTML will be familiar with the
+structure of XML whereby "tags" enclosed in angle brackets denote that a
+particular piece of data will follow. Let us examine the follow example:
+
+.. code-block:: xml
+
+    <person>
+      <firstname>John</firstname>
+      <lastname>Smith</lastname>
+      <age>27</age>
+      <occupation>Health Physicist</occupation>
+    </person>
+
+Here we see that the first tag indicates that the following data will describe a
+person. The nested tags *firstname*, *lastname*, *age*, and *occupation*
+indicate characteristics about the person being described.
+
+In much the same way, OpenMC input uses XML tags to describe the geometry, the
+materials, and settings for a Monte Carlo simulation. Note that because the XML
+files have a well-defined structure, they can be validated using the
```

Comment:
This might be a good segue into the subsection on input validation within "nXML", what do you think? Otherwise, it seems a little out of place at the bottom of the installation page, though I can understand the original reason for placing it there.